viewer/packages/api/src/public/RevealOptions.ts

```diff
@@ -18,6 +18,7 @@ export type RevealOptions = {
   continuousModelStreaming?: boolean;
   outputRenderTarget?: { target: THREE.WebGLRenderTarget; autoSize?: boolean };
   rendererResolutionThreshold?: number;
+  movingResolutionFactor?: number;
```

Comment:
Suggest this is renamed to `cameraResolutionFactor`. Also, add TSDoc - I also see that most other stuff here is missing and that we document a non-existant field.

Reply:
Or maybe this is purely internal? Should still be documented, but not that important

Reply:
You sure about the `cameraResolutionFactor` name? I believe it should somehow notify that we're talking about the resolution when the camera is moving.